core/pom.xml

```diff
@@ -30,7 +30,8 @@
   <url>https://www.eclipse.org/hono</url>
 
   <properties>
-    <java.level>1.8</java.level>
+    <main.java.level>1.8</main.java.level>
+    <test.java.level>11</test.java.level>
```

Comment:
I do not think that this is the solution to the problem. The underlying problem FMPOV is that the `HonoTimestamp` annotation is not implemented correctly and, coincidentally, works on Java 11 but doesn't on Java 8. I'd rather fix the underlying issue than addressing the symptoms only. Let my dig into this a little more ...

Reply:
ok

Reply:
@Alfusainey 

I have pushed a change to master which should fix the problem with the `HonoTimestamp` annotation. After rebasing, you should be able to run your test case in core without any further modifications of the JVM being used ...

Reply:
nice 👍 i will undo this last commit and force push to this branch